src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        System.out.println(new Lottery().getRandomBall());
+        System.out.println(new Lottery().getRandomBall());
+        System.out.println(new Lottery().getRandomBall());
```

Comment:
don't create a new Lottery object every time before calling getRandomBall, and better use loop
```suggestion
        System.out.println(new Lottery().getRandomBall());
        System.out.println(new Lottery().getRandomBall());
        System.out.println(new Lottery().getRandomBall());
```